### PR TITLE
Fix Issue #37

### PR DESCRIPTION
**Pull request by leo-dor**
Fix Issue #37 -- allow to clone properly on Windows.
* Renamed all test files to replace '?' with '_'
* Changed FileSystemFetcher to likewise replace '?' with '_' when loading a test file
* Added logic to FileSystemFetcher to check for all permutations of parameters contained in a test file's name.
* Removed symlinks from repo.
* The repo now clones cleanly on Windows.
* Unit tests passed.

### CODE REVIEW

File: discogs_client/fetchers.py [hunk 1]
```diff
@@ -101,13 +102,54 @@ def fetch(self, client, method, url, data=None, headers=None, json=True):
...
         path = os.path.join(self.base_path, base_name)
+
+        # The exact path might not exist, but check for files with different
+        # permutations of query parameters.
+        if not os.path.exists(path):
+            base_name = self.check_alternate_params(base_name, json)
+            path = os.path.join(self.base_path, base_name)
+
         try:
+            path = path.replace('?', '_')  # this fixes Issue 37
```
Comment (rodneykeeling) [inline discogs_client/fetchers.py]: No need for this comment here.

File: discogs_client/fetchers.py [hunk 2]
```diff
@@ -101,13 +102,54 @@ def fetch(self, client, method, url, data=None, headers=None, json=True):
...
             with open(path, 'r') as f:
                 content = f.read().encode('utf8')  # return bytes not unicode
             return content, 200
         except:
             return self.default_response
 
+    def check_alternate_params(self, base_name, json):
+        """ parse_qs() result is non-deterministic - a different file might be
```
Comment (rodneykeeling) [inline discogs_client/fetchers.py]: We generally add a line break on the first line here. e.g.,
```
"""
parse_qs() result is non-deterministic - a different file might be
```

**Comment by leo-dor** [comment]
Thanks for the feedback, comments fixed.

**Comment by rodneykeeling** [comment]
Thanks, this looks good to me. I'll see if I can have another developer take a second look as well.

:+1:

**Comment by vreon** [comment]
LGTM :+1:

**Pull request merged**